### PR TITLE
textarea getting expanded more than container

### PR DESCRIPTION
**Pull request by nvnvenki**
The issue is when you resize the text area(drag from right bottom) It goes out of the container. Fix for this is having resize:vertical  for the text area.

### CODE REVIEW

**Comment by nvnvenki** [comment]
Example of issue
http://jsfiddle.net/nvnvenki/f4jkj1p0/

when you resize the text area(drag from right bottom) It goes out of the column.

**Comment by acshef** [comment]
As far as resizable fields go, I believe using `resize:vertical` semantically fits better for the result you're after instead of using `max-width:100%`

**Comment by nvnvenki** [comment]
Thanks @acshef.

I agree. Changed the fix accordingly.

File: less/forms.less [hunk 1]
```diff
@@ -153,6 +153,10 @@ output {
   // Reset height for `textarea`s
   textarea& {
     height: auto;
+    resize: vertical; // fix to stop horizontal resize.
+    &[disabled] {
```
Comment (cvrebert) [inline less/forms.less]: Why is this necessary?

**Comment by cvrebert** [comment]
This seems too opinionated/harsh to me.

**Comment by patrickhlauke** [comment]
Agree...some users will see the ability to resize the textarea how they please (regardless of site layout) as a feature, rather than a bug. IMHO of course.

**Comment by nvnvenki** [comment]
Is it not bad to allow text area to expand more than it container? Sorry If this was the expected behvaiour. Other UI frameworks such as semantic UI, foundation has this.

**Comment by patrickhlauke** [comment]
"bad" is subjective...some users will see it as bad that if it does expand more than the container, while other users will see it as bad if it doesn't.

up to @mdo to make a judgement call here, I guess

**Comment by nvnvenki** [comment]
Hey,
Looks like github is not letting its comment text area to expand more than its container horizontally by adding  resize: veritcal; :) This was not the case before at least till last month If I remember properly :)

**Comment by mdo** [comment]
GitHub has been doing this for awhile now—it's certainly been that way for since at least April of last year (according to `git blame`). We won't be doing this though for the time being. Maybe in v4.

Thanks though!
